cranelift/isle/isle/src/lexer.rs

```diff
@@ -232,7 +232,7 @@ impl<'a> Lexer<'a> {
                 debug_assert!(!s.is_empty());
                 Ok(Some((start_pos, Token::Symbol(s.to_string()))))
             }
-            c if (c >= b'0' && c <= b'9') || c == b'-' => {
+            c @ b'0'..=b'9' | c @ b'-' => {
```

Comment:
Does Rust allow using the following syntax instead, and would you agree it's easier to read?
```suggestion
            c @ (b'0'..=b'9' | b'-') => {
```

Reply:
It does. Changed it.